src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -82,11 +82,14 @@ def __init__(
         copy: bool = False,
         name: object = None,
         tupleize_cols: bool = True,
+        convert_to_lazy: bool = True,
```

Comment:
can we add comments for all init arguments ?

Reply:
even the native pandas ones?